jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/CaseLogCleanupCommandTest.java

```diff
@@ -0,0 +1,256 @@
+        logger.info("CASE CREATED: {}", caseIdComplete);
+        String caseIdIncomplete = caseService.startCase(DEPLOYMENT_UNIT, LOG_INCOMPLETE_CLEANUP_ID);
+        logger.info("CASE CREATED: {}", caseIdIncomplete);
+
+        // Advance time 1+ second forward
+        Thread.sleep(1010);
+
+        // Verify presence of data
+        Assertions.assertThat(getCaseFileDataLogSize()).isEqualTo(2);
+        Assertions.assertThat(getCaseAssignmentRoleLogSize()).isPositive();
+
+        // Schedule cleanup job
+        scheduleLogCleanup(null, "1s", LOG_CLEANUP_ID);
+        countDownListener.waitTillCompleted();
+
+        // Verify absence of data
+        Assertions.assertThat(getCaseFileDataLogSize()).isEqualTo(1);
+        Assertions.assertThat(getCaseAssignmentRoleLogSize()).isPositive();
+    }
+
+    // ------------------------ Helper Methods ------------------------
+
+
+
+    protected ExecutorService getExecutorService() {
+        if (executorService == null) {
+            executorService = ExecutorServiceFactory.newExecutorService(getEmf());
+            executorService.setThreadPoolSize(executorThreads);
+            executorService.setRetries(executorRetries);
+            executorService.setInterval(executorInterval);
+            executorService.init();
+
+            logger.debug("Created ExecutorService with parameters: '" + executorThreads + " threads', '" + executorRetries + " retries', interval '" + executorInterval + "s'");
+        }
+        return executorService;
+    }
+
+    private long getCaseAssignmentRoleLogSize() {
+        EntityManager em = this.getEmf().createEntityManager();
+        long count = (long) em.createQuery("SELECT COUNT(O) FROM CaseRoleAssignmentLog O").getSingleResult();
+        em.close();
+        return count;
+    }
+
+    private long getCaseFileDataLogSize() {
+        EntityManager em = this.getEmf().createEntityManager();
+        long count = (long) em.createQuery("SELECT COUNT(O) FROM CaseFileDataLog O").getSingleResult();
+        em.close();
+        return count;
+    }
+
+    private void scheduleLogCleanup(Date olderThan,
+                                    String olderThanDuration,
+                                    String forDeployment) {
+        CommandContext commandContext = new CommandContext();
+        commandContext.setData("EmfName", "org.jbpm.test.persistence");
+
+        commandContext.setData("SingleRun", "true");
+        commandContext.setData("DateFormat", dateFormat);
+        if (olderThan != null) {
+            commandContext.setData("OlderThan", new SimpleDateFormat(dateFormat).format(olderThan));
+        }
+        if (olderThanDuration != null) {
+            commandContext.setData("OlderThanPeriod", olderThanDuration);
+        }
+        commandContext.setData("ForDeployment", forDeployment);
+        getExecutorService().scheduleRequest("org.jbpm.casemgmt.impl.audit.CaseLogCleanupCommand", commandContext);
```

Comment:
`DateFormat`, `OlderThan`, `OlderThanPeriod` and `ForDeployment` are ignored during the execution I guess.